src/zenml/integrations/mlflow/model_registries/mlflow_model_registry.py

```diff
@@ -362,10 +362,16 @@ def register_model_version(
 
         Raises:
             RuntimeError: If the registered model does not exist.
+            ValueError: If no model source URI was provided.
 
         Returns:
             The registered model version.
         """
+        if not model_source_uri:
```

Comment:
@safoinme Mlflow now requires a model source URI when creating a model version. Does it make sense to fail here or should we use some default value instead?

Reply:
it must fail if it doesn't exist